astro/src/content/docs/apis/system.mdx

```diff
@@ -225,25 +225,45 @@ This API is used to retrieve the status of a reindex request. This may be useful
 
 The Status API is used to retrieve the current status and metrics for FusionAuth. This is useful for monitoring. For health checks, prefer [the Health API](/docs/apis/system#retrieve-system-health).
 
-By default, this API requires authentication. If you prefer to allow unauthenticated access to this endpoint from local scrapers, you may set `fusionauth-app.local-metrics.enabled=true`. See the  [configuration reference](/docs/reference/configuration) for more info.
+By default, this API requires authentication to return full results. If you prefer to allow unauthenticated access to this endpoint from local scrapers, you may set `fusionauth-app.local-metrics.enabled=true`. See the [configuration reference](/docs/reference/configuration) for more info.
+
+If you call this API without an API key, you will receive a status response only.
 
 <Aside type="note">
 FusionAuth also supports a system status check [using Prometheus](/docs/operate/monitor/prometheus).
 </Aside>
 
 ### Request
 
-<API method="GET" uri="/api/status" authentication={["local-bypass", "none"]} title="Return the system status without an API key"/>
+<API method="GET" uri="/api/status" authentication={["none"]} title="Return the system status without an API key"/>
 
 <API method="GET" uri="/api/status" authentication={["api-key"]} title="Return the system status with an API key"/>
```

Comment:
Note that we do also accept `"basic"` here. Same as the prometheus 

```
<API method="GET" uri="/api/prometheus/metrics" authentication={["basic", "api-key"]} title="Retrieve FusionAuth application metrics to use with Prometheus"/>

```

The status api, version api and metrics API all accept this, but it looks like we have only documented it for the Prometheus metrics API.